purchase.adoc

```diff
@@ -388,288 +517,533 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
         "<https://purchase.izettle.com/purchases/v2?limit=10&descending=true&lastPurchaseHash=1485858080407AOjsYOefEeax5pDdmnaGXw>; rel=\"next\""
   ]
 }
-```
+----
+--
+
+'''
+
+====
+
+==== Fetch purchase by id
+
+`GET /purchase/v2/{purchaseUuid}`
+
+[%collapsible]
+====
+Returns a purchase with given UUID.
+
+*Parameters*
+[cols="15%,10%,10%,10%,55%"]
+|===
+|Name|Type|In|Required/Optional|Description
+
+|purchaseUuid|string|path|required|Unique id of the purchase as UUID within iZettle system.
+|===
+
+*Responses*
 
-### Get purchase details
+`Status: 200 OK` - Successful, response json is returned. +
+`Status: 404 Not found` - Purchase with given id not found. +
```

Comment:
Purchase with the given ID is not found.

Reply:
No

Reply:
This is only a human readable explanation of HTTP status code returned by the API.

Reply:
I see. I was just reminded of other crafted UX text for 404 error code.